row_test.go

```diff
@@ -34,16 +34,18 @@ func TestGettingRowValues(t *testing.T) {
 
 	rows, err = crud.GetRowValuesOf(Post{})
 	assert.Nil(t, err)
-	assert.Equal(t, len(rows), 3)
+	assert.Equal(t, len(rows), 4)
```

Comment:
Is this `4` now because of the new column in it is added?

Reply:
yep I think so
![image](https://user-images.githubusercontent.com/6832539/158177933-69701254-f267-4783-b3e0-cb050def3d4f.png)
though I'm not sure why it wasn't 4 in the first place